konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/api/context/TraceEvent.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  ******************************************************************************
+ *  * Copyright (c) 2020 Konduit K.K.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Apache License, Version 2.0 which is available at
+ *  * https://www.apache.org/licenses/LICENSE-2.0.
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  * License for the specific language governing permissions and limitations
+ *  * under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *  *****************************************************************************
+ */
+package ai.konduit.serving.pipeline.api.context;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class TraceEvent {
+    public static enum EventType {
+        START, END;
+    }
+    private String name;
+    private List<String> cat;
+    private long ts;
+    private long pid;
+    private long tid;
+    EventType ph;
+}
```

Comment:
Two more small change needed to comply with the Chrome profiler trace format:
(a) cat - should be a String (comma separated) not a List<String> - that will produce JSON like `["a", "b"]` not `"a,b"`
(b) EventType enum - needs to be "B" and "E" - (see table at end of page 4 - https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/edit )

Also access modifier - `private EventType ph`